Sources/CodeEditTextView/Controller/STTextViewController.swift

```diff
@@ -127,7 +127,10 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
 
         rulerView = STLineNumberRulerView(textView: textView, scrollView: scrollView)
         rulerView.backgroundColor = useThemeBackground ? theme.background : .clear
-        rulerView.textColor = .systemGray
+        rulerView.textColor = .systemGray.blended(
+            withFraction: 0.5,
+            of: .darkGray
+        ) ?? .darkGray
```

Comment:
I'd declare a private var in which computes this.

Somewhere at the top of the `class`:
```swift
private var rulerTextColor: NSColor {
    return .systemGray.blended(
        withFraction: 0.5
        of: .darkGray
    ) ?? .darkGray
}
```

This way it can be accessed anywhere in `STTextViewController`

Reply:
Please also make sure that it looks good in both dark and light mode with both dark and light themes. Also append screenshots for each case with a comparison to the same setup in Xcode.

Reply:
I can verify it looks good in both light and dark mode, I'll show a screenshot

Reply:
i just added a screenshot